ml-data-hub-plugin/src/main/groovy/com/marklogic/gradle/task/ClearUserArtifactsTask.groovy

```diff
@@ -0,0 +1,14 @@
+package com.marklogic.gradle.task
+
+import com.marklogic.hub.impl.DataHubImpl
+
+class ClearUserArtifactsTask extends AbstractConfirmableHubTask {
+
+    @Override
+    void executeIfConfirmed() {
+        println "Clearing user artifacts. This requires a user with at least 'data-hub-developer' role."
```

Comment:
I don't think this is technically true - a user just needs update access to each artifact type. I think that's the better message to give - "Clearing user artifacts. This requires a user that has update permission on each artifact type."